src/state/workload.rs

```diff
@@ -710,7 +788,7 @@ impl WorkloadStore {
 
     /// Finds the workload by address, as an arc.
     pub fn find_address(&self, addr: &NetworkAddress) -> Option<Arc<Workload>> {
-        self.by_addr.get(addr).cloned()
+        dbg!(dbg!(self.by_addr.get(addr)).map(WorkloadByAddr::get))
```

Comment:
For my understanding: what does this macro do, and why do we call it twice?

Reply:
its for debugging - shouldn't merge! but it basically dumps the value to stderr with some line info, etc